pgbot/commands/__init__.py

```diff
@@ -104,18 +105,25 @@ async def handle(invoke_msg: discord.Message, response_msg: discord.Message = No
         )
 
     if not common.TEST_MODE and not common.GENERIC:
-        await embed_utils.send_2(
-            common.log_channel,
-            title=f"Command invoked by {invoke_msg.author} / {invoke_msg.author.id}",
-            description=discord.utils.escape_markdown(invoke_msg.content),
-            fields=(
-                (
-                    "\u200b",
-                    f"by {invoke_msg.author.mention}\n**[View Original]({invoke_msg.jump_url})**",
-                    False,
+        log_txt_file = None
+        escaped_cmd_text = discord.utils.escape_markdown(invoke_msg.content)
+        if len(escaped_cmd_text) > 2047:
+            with io.StringIO(invoke_msg.content) as log_buffer:
+                log_txt_file = discord.File(log_buffer, filename="command.txt")
+
+        await common.log_channel.send(
+            embed=embed_utils.create(
+                title=f"Command invoked by {invoke_msg.author} / {invoke_msg.author.id}",
+                description=escaped_cmd_text if len(escaped_cmd_text) <= 2047 else escaped_cmd_text[:2048]+"...",
```

Comment:
The 2048 on this line should be 2045, because those extra `...` add to the output too

Reply:
🤦 right